networkdb/networkdb.go

```diff
@@ -328,6 +328,9 @@ func (nDB *NetworkDB) GetEntry(tname, nid, key string) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
+	if entry != nil && entry.deleting {
+		return nil, types.NotFoundErrorf("entry not found in table %s with network id %s and key %s", tname, nid, key)
```

Comment:
Masking the existence into a "not found" message may be confusing if this message makes into logs but say the GC does not trigger for a while. How about making the message something along the lines of: "entry found but in deleting state. returning not found" to keep things super clear for support engineers?

Reply:
changing in: `return nil, types.NotFoundErrorf("entry in table %s network id %s and key %s deleted and pending garbage collection", tname, nid, key)`